### DISCUSSION TITLE
AutoComplete - displaying selected value in the input as a placeholder

### DISCUSSION THREAD

**Discussion by martin-tests** [primefaces/community#2421] [PrimeVue]
Currently when clicking on AutoComplete (non-multiple) input, the selected option is there as a filled-in editable input value:
![image](https://github.com/user-attachments/assets/28f001b3-954e-4e8c-8dde-7683d079942d)

Is there any way to do it so that the selected option is not displayed in the input but as a placeholder, whereas the model still retains that e.g. "Option 1" is the selected option?
![image](https://github.com/user-attachments/assets/f4f333b5-6fa3-452f-aa2f-d0d7fc99f388)

**Comment by ibrahimkelly** [comment]
> Currently when clicking on AutoComplete (non-multiple) input, the selected option is there as a filled-in editable input value: ![image](https://private-user-images.githubusercontent.com/50135817/353477085-28f001b3-954e-4e8c-8dde-7683d079942d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI4NTk0MzMsIm5iZiI6MTcyMjg1OTEzMywicGF0aCI6Ii81MDEzNTgxNy8zNTM0NzcwODUtMjhmMDAxYjMtOTU0ZS00ZThjLThkZGUtNzY4M2QwNzk5NDJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA1VDExNTg1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk5NDU2ZjFiMzIwM2E3NjNiZWYyYjYyNTY0MDJjNTUyOGE2YTI4MTU5ZWZkNzExYzMzNmNhZTE2ZDMxYzE5MzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UsHZIzNAu7Gu8mi_SwTTHFsDs8W5cY6AApvhobbh5xU)
> 
> Is there any way to do it so that the selected option is not displayed in the input but as a placeholder, whereas the model still retains that e.g. "Option 1" is the selected option? ![image](https://private-user-images.githubusercontent.com/50135817/353479536-f4f333b5-6fa3-452f-aa2f-d0d7fc99f388.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI4NTk0MzMsIm5iZiI6MTcyMjg1OTEzMywicGF0aCI6Ii81MDEzNTgxNy8zNTM0Nzk1MzYtZjRmMzMzYjUtNmZhMy00NTJmLWFhMmYtZDBkN2ZjOTlmMzg4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA1VDExNTg1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY0ZjY4ZmZjMTFhMjExMTZmMDBlYWVjNmI0Y2NhZGFhNTBlZmMxODdiYTFmMGU4ODFhZmFkY2Q3ZTY1YjRiMTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1_b-VZ_1FBWAQZymI9So4VRoN3DGBCVgO8tEXjSTXio)

I thought this is not related to PrimeNG; you can handle it on the Angular side. This leads to another question: if clicking results in a placeholder, how would you set the value?
